resource/reapi/bindings/go/src/fluxmodule/reapi_module_test.go

```diff
@@ -0,0 +1,28 @@
+/*****************************************************************************\
+ * Copyright 2023 Lawrence Livermore National Security, LLC
+ * (c.f. AUTHORS, NOTICE.LLNS, LICENSE)
+ *
+ * This file is part of the Flux resource manager framework.
+ * For details, see https://github.com/flux-framework.
+ *
+ * SPDX-License-Identifier: LGPL-3.0
+\*****************************************************************************/
+
+package fluxmodule
+
+import (
+	"fmt"
+	"io/ioutil"
+	"testing"
+)
+
+func TestFluxmodule(t *testing.T) {
+	ctx := NewReapiModule()
+	jobspec, err := ioutil.ReadFile("/root/flux-sched/t/data/resource/jobspecs/basics/test001.yaml")
+	if ctx == nil {
+		t.Errorf("Context is null")
```

Comment:
```suggestion
		t.Errorf("context is null")
```
reference https://github.com/golang/go/wiki/CodeReviewComments#error-strings